apps/vaporgui/VDoubleRangeMenu.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#include "VNumericFormatMenu.h"
+
+class VDoubleLineEditAction;
+
+//! \class VIntRangeMenu
```

Comment:
Wrong documentation for VDoubleRangeMenu class?

Reply:
Thanks John.  Fixed.